public/editor/scripts/project-list.js

```diff
@@ -22,6 +22,18 @@ require(["jquery", "constants", "analytics", "moment"], function($, Constants, a
   var username = encodeURIComponent($("#project-list").attr("data-username"));
   var locale = $("html")[0].lang;
   var queryString = window.location.search;
+  var isLocalStorageAvailable = !!(window.localStorage);
+  var favorites;
+  if(isLocalStorageAvailable){
+    try {
+      favorites = JSON.parse(localStorage.getItem("project-favorites"));
+      if(favorites === null) {
```

Comment:
I initially tried to use `var favorites = [];` but if project-favorites does not exist in localStorage and localStorage is available, favorites gets filled with `null` instead of an empty array. I could use `var favorites = [];` if i added another condition to the if statement such as `if(isLocalStorageAvailable && localStorage.getItem("project-favorites")  !== null)`.
Would there be a better way around this?